packages/scandipwa/src/component/Field/Field.component.js

```diff
@@ -202,16 +202,20 @@ export class Field extends PureComponent {
             ...events,
             onChange: onChange || noopFn
         };
+        // if button value is "none" do not disable
+        const disableButton = (!value.match('none') && isDisabled);
+        const isChecked = disableButton ? !isDisabled : null;
```

Comment:
Since will most lucky remove default value for checked I suggest to rename disableButton to isButtonDisabled. We consider to use prefix 'is' for booleans.

Reply:
Done